README.md

```diff
@@ -167,7 +167,7 @@ class E2eTest extends PantherTestCase
     {
         $symfonyClient = static::createClient(); // A cute kitty: Symfony's functional test tool
         $goutteClient = static::createGoutteClient(); // An agile lynx: Goutte
-        $pantherClient = static::createPantherClient(); // A majestic Panther
+        $pantherClient = static::createPantherClient(); // A majestic Panther (defaults to a Chrome client)
```

Comment:
Does the original `A magestic Panther` comment still add anything?

Should this part be removed?

Reply:
I find it funny, removing it destroys the whole cat analogy thing :)